Tools/autotest/common.py

```diff
@@ -3773,6 +3773,9 @@ def add_param_value(mav, m):
                             self.context_get().parameters.append((name, original_values[name]))
                     received.add(name)
                     continue
+                if original_values[name] == autopilot_values[name] and i % 2 == 0:
+                    self.send_get_parameter_direct(name)
+                    self.progress("Requesting (%s) (retry=%u)" % (name, i))
```

Comment:
Why?  Needs a comment.  I'm doubtful this is required.

I don't really want to change the autotest suite to allow for really lossy links if that's what htis is about.

Reply:
I am connected on serial2 on SITL and it fails to get  the set response, so I need to explictly ask for a get